protos/agent_worker.proto

```diff
@@ -2,15 +2,11 @@ syntax = "proto3";
 
 package agents;
 
-option csharp_namespace = "Microsoft.AutoGen.Contracts";
+option csharp_namespace = "Microsoft.AutoGen.Protobuf";
```

Comment:
the namespace Microsoft.AutoGen.Protobuf is odd - leaks an implementation detail that could change in the future. can we use something that describes the generated classes?